mp-istio-sample/serviceA/pom.xml

```diff
@@ -0,0 +1,334 @@
+<?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>projects</groupId>
+    <artifactId>mp-istio-sample-servicea</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>war</packaging>
+
+    <!-- contains default pluginManagement configuration for liberty-maven-plugin -->
+    <parent>
```

Comment:
Anything Liberty specific should be in a separate profile to make it easy for other implementations to add them as an option